# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.h
# Commit: 40fcc0cdc593
# Full Hash: 40fcc0cdc593fa337273631de435a2901ec881f6
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-02-07 04:17:40
# Regressor Bug: 1878286
# File Overlap Count: 2
# Description:
#   Bug 1878286 - Eliminate usage of fontlist::Pointer in IPDL. r=gfx-reviewers,lsalzman
#   
#   No change in behavior.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D200564
# ==============================================================================

diff -r c48ca61bc925 -r 40fcc0cdc593 dom/ipc/ContentParent.h
--- a/dom/ipc/ContentParent.h	Tue Feb 06 10:15:29 2024 +0000
+++ b/dom/ipc/ContentParent.h	Tue Feb 06 10:41:50 2024 +0000
@@ -1122,17 +1122,19 @@
                                                const uint32_t& aFamilyIndex,
                                                const bool& aLoadCmaps);
 
-  mozilla::ipc::IPCResult RecvSetCharacterMap(
-      const uint32_t& aGeneration, const mozilla::fontlist::Pointer& aFacePtr,
-      const gfxSparseBitSet& aMap);
+  mozilla::ipc::IPCResult RecvSetCharacterMap(const uint32_t& aGeneration,
+                                              const uint32_t& aFamilyIndex,
+                                              const bool& aAlias,
+                                              const uint32_t& aFaceIndex,
+                                              const gfxSparseBitSet& aMap);
 
   mozilla::ipc::IPCResult RecvInitOtherFamilyNames(const uint32_t& aGeneration,
                                                    const bool& aDefer,
                                                    bool* aLoaded);
 
-  mozilla::ipc::IPCResult RecvSetupFamilyCharMap(
-      const uint32_t& aGeneration,
-      const mozilla::fontlist::Pointer& aFamilyPtr);
+  mozilla::ipc::IPCResult RecvSetupFamilyCharMap(const uint32_t& aGeneration,
+                                                 const uint32_t& aIndex,
+                                                 const bool& aAlias);
 
   mozilla::ipc::IPCResult RecvStartCmapLoading(const uint32_t& aGeneration,
                                                const uint32_t& aStartIndex);